core/src/main/java/org/kiwix/kiwixmobile/core/main/CoreMainActivity.java

```diff
@@ -1114,7 +1111,7 @@ private boolean hasPermission(String permission) {
   private void requestExternalStoragePermission() {
     ActivityCompat.requestPermissions(
       this,
-      new String[] { Manifest.permission.READ_EXTERNAL_STORAGE },
```

Comment:
These spaces should not have changed, have you tried reformatting the file with `ctrl+alt+l`?

Reply:
That is what I did originally, which I tried to undo in my last commit. Will fix!

Reply:
Move it back to one line, press `ctrl+alt+l` again and see if it is being a moody git.

My formatter doesn't make any noise about it so make sure you have no changes to the codestyle file under the `idea` folder

Reply:
I meant what @yashk2000 mentioned. That all formatting should be undone because it is not relevant to the issue. I tried removing the formatting done in #1885 Some refactoring for easier overview of handleIntentActions method with the latest commit. `ctrl + alt + l` is working but I undid it because it was requested.

Reply:
We all share the same formatter, your formatter shouldn't be changing things if mine isn't.

Currently there are spaces around the `{ }` and I don't want that to change.

All files in the project are correctly formatted so if you run the formatter and some line you haven't touched changes its formatting then this points to an issue on your end

Reply:
I have `format,optimise,indent,save all` all bound to a macro bound to `ctrl+s` and I hit that probably every 20 seconds out of insane habit

Reply:
ah, ok. I'll look into it. Thanks for clarifying. Binding those to ctrl+s seems nice!

Reply:
Alright, I had to manually set the style to SquareAndroid.. Now it should be correct!

Reply:
The project style is imported automatically, it is in git in our repo under the `idea` folder. This is quite confusing

Reply:
@macgills Might be with how I imported the project. I believe I imported the Core module only, which works but it might have missed things such as the stylings.

Reply:
hmm maybe re-import the whole thing but as long as your codestyle doesn't conflict it should be okay